### ISSUE TITLE
Is epoch = 5000  too large?

### ISSUE THREAD

**Issue by lightwithshadow** [onejiin/CycleGAN-VC2#3]
Thank you for your contribution to this project. 
       When I was running demo, an epoch take about 30 minutes. Is epoch = 5000 too large?
 Looking forward to your reply!  Thanks!!!

**Comment by onejiin** [comment]
Hi,

I think 5000epoch are quite large, thus overfitting may be occurring.
Code includes tensorboard as "--tensorboard_log_dir".
I don’t know the database what you use, but when validation set inferences, learning rate and training epoch can be adjusted by checking the MSE curve.
For VCC2016 database, I will update the learning curve.

Cheers,
Wonjin

**Comment by lightwithshadow** [comment]
Hi,


Thanks for your reply! I will try it again.&nbsp;Thanks for your attention to this matter.


Cheers,
shadow




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Wonjin, Jung"<notifications@github.com&gt;;
发送时间:&nbsp;2019年9月23日(星期一) 中午1:25
收件人:&nbsp;"onejiin/CycleGAN-VC2"<CycleGAN-VC2@noreply.github.com&gt;;
抄送:&nbsp;"1101174181"<1101174181@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [onejiin/CycleGAN-VC2] Is epoch = 5000  too large? (#3)




Hi,
 
I think 5000epoch are quite large, thus overfitting may be occurring.
 Code includes tensorboard as "--tensorboard_log_dir".
 I don’t know the database what you use, but when validation set inferences, learning rate and training epoch can be adjusted by checking the MSE curve.
 For VCC2016 database, I will update the learning curve.
 
Cheers,
 Wonjin
